src/accelerate/components/List.tsx

```diff
@@ -97,16 +131,39 @@ export default function List( props: Props ) {
 							} }
 						/>
 					</div>
+					<div className="table-add-new">
+						<Dropdown
+							className=""
+							contentClassName=""
+							position="bottom right"
+							renderToggle={ ( { isOpen, onToggle } ) => (
+								<Button
+									variant="primary"
+									onClick={ onToggle }
+									aria-expanded={ isOpen }
+									className='dashicons-before dashicons-plus'
+								>
+									{ __( 'Add New', 'altis' ) }
+								</Button>
+							) }
+							renderContent={ ( { onClose } ) => (
+								<MenuGroup>
+									{/* <MenuItem onClick={ onAddNew( 'block' ) }>
+										{ __( 'Block', 'altis' ) }
+									</MenuItem> */}
+									<MenuItem onClick={ onAddNew( 'post' ) }>
+										{ __( 'Post', 'altis' ) }
+									</MenuItem>
+									<MenuItem onClick={ onAddNew( 'page' ) }>
```

Comment:
Can we list all post types rather than hard coding? Not sure if there was any discussion / annotation to that effect. cc @noeltock

Reply:
We didn't have any conversation about which types to allow, but I guess we can detect types and populate the list accordingly 👍

Reply:
Already being passed to the JS via localise script

Reply:
Yeah, was just using it, and it does seem you can create a reusable block via the URL.

Reply:
You can in Altis so might be because of the feature to show blocks in the admin. The arguments used when the post type is registered might need to be tweaked / filtered to allow it but `?post-new / edit.php?post_type=wp_block` should work.